array-manipulation-3/src/zip.js

```diff
@@ -0,0 +1,15 @@
+/* exported zip */
+function zip(first, second) {
+  var shortArr = first;
+  var combinedArr = [];
+  if (second.length < first.length) {
+    shortArr = second;
+  }
+  for (var i = 0; i < shortArr.length; i++) {
+    var miniArr = [];
+    miniArr.push(first[i]);
+    miniArr.push(second[i]);
```

Comment:
You can pass multiple items to `push` so you only need to call push once here.

For example:
```JAVASCRIPT
arr.push('Item 1', 'Item 2', 'Item 3');
```